pkg/endpoint/endpoint.go

```diff
@@ -1660,6 +1660,9 @@ func (e *Endpoint) HasLabels(l pkgLabels.Labels) bool {
 // replaceInformationLabels replaces the information labels of the endpoint.
 // Must be called with e.Mutex.Lock().
 func (e *Endpoint) replaceInformationLabels(l pkgLabels.Labels) {
+	if l == nil {
```

Comment:
I was worried that this changes behavior as nil no longer removes all labels but I have not found a caller that previously passed in nil.

Please add a comment to the function that indicates that if l is nil, no update is performed.